### DISCUSSION TITLE
Quale mini-pc ?

### DISCUSSION THREAD

**Discussion by guestisp** [indomus/forum#2473] [Hardware & Firmware]
Domanda ricorrente, lo so, ma non saprei proprio quale scegliere.
Preferirei fanless, preferirei sotto i 200 euro
Immagino che doppia rete o rete+wifi con antenna esterna sia impensabile in questa fascia di prezzo.
Dovrà gestire alcune telecamere con motioneye

Idee ?

**Comment by indomus** [comment]
> Domanda ricorrente, lo so, ma non saprei proprio quale scegliere. Preferirei fanless, preferirei sotto i 200 euro Immagino che doppia rete o rete+wifi con antenna esterna sia impensabile in questa fascia di prezzo. Dovrà gestire alcune telecamere con motioneye
> 
> Idee ?

Quelli [qui in fondo](https://indomus.it/componenti/mini-pc-intel-nuc-e-similari/) li hai visti?

**Comment by guestisp** [comment]
si. ho letto le recensioni ma sono già di qualche anno fa, magari c'è qualcosa di più recente. ad esempio il Nipogi jk06 che è totalmente fanless e con un celeron più performante, ma non ho trovato nessuna recensione online

chiedevo apposta dei consigli, qualche idea?

**Reply by indomus** [reply]
mah, veramente l'elenco lo aggiorniamo settimana dopo settimana, non sta fermo. [questo](https://amzn.to/3oxZhvb) è una discreta bestia, altrimenti anche [questo](https://amzn.to/45mEgV1) va molto bene.

**Reply by guestisp** [reply]
Io ho guardato quelli dove era disponibile una recensione.
[Questo](https://amzn.to/431PiNF) come ti sembra ?

**Reply by indomus** [reply]
fanless, espandibile, buon processore, ben valutato, dovrebbe esser buono

**Reply by guestisp** [reply]
> fanless, espandibile, buon processore, ben valutato, dovrebbe esser buono

Grazie.
Come telecamere invece hai suggerimenti ? Mi piacerebbe avere 1 modello relativamente compatto da posizionare interno casa (se poi lo stesso modello fosse usabile anche all'esterno, tanto meglio). ***FONDAMENTALE*** che non funzioni esclusivamente via cloud, cosa che terrò bloccato, quindi RTSP, ONVIF. La Imou bullet mi ispira e sembra compatta ma mi sembra di capire che solo il video è usabile da rete locale, i comandi impartiti da HA devono passare per il cloud..... Alternative ? (ho già una Tapo C500 in negozio che reputo ottima e funziona perfettamente via LAN, ma è solo 1080)

**Reply by indomus** [reply]
ne abbiamo [testate svariate](https://indomus.it/componenti/#ipcam), vedi l'elenco

**Reply by guestisp** [reply]
> ne abbiamo [testate svariate](https://indomus.it/componenti/#ipcam), vedi l'elenco

ecco questo elenco non l'avevo visto. Grazie.

Mi sembra di capire che forse la Foscam (la X4, quella da 4MP) sia una delle più interessanti, si può configurare da zero senza nemmeno installare l'app, è piccolissima, supporta nativamente onvif e rtsp e costa poco.... Non conosco minimamente il brand quindi le avevo saltate a piedi pari.
